.mend/notes/xbrlkit-scout-report.md

```
@@ -0,0 +1,276 @@
+| Missing Step Handlers | ~45 | High |
+| Missing Plans | Multiple | Low |
+
+---
+
+## 1. Unactivated Scenarios 🔕
+
+**Count:** 40 scenarios lack `@alpha-active` tag
+
+Scenarios with `@SCN-*` tags but **WITHOUT** `@alpha-active`:
+
+### Foundation (4 scenarios)
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)